services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteSpec.groovy

```diff
@@ -388,6 +388,36 @@ class AutoRerouteSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    def "Flow is not rerouted when one of the flow endpoints goes down"() {
```

Comment:
'flow endpoints' sounds a bit misleading to me. If I didn't see the test, I would have thought that the src/dst switch goes down.

Reply:
fixed.